packages/sdk/src/__tests__/__snapshots__/handlePanic.test.ts.snap

```diff
@@ -2,7 +2,6 @@
 
 exports[`handlePanic when sendPanic fails, the user should be alerted by a reportFailedMessage 1`] = `
 "Oops, an unexpected error occured!
-test-message
```

Comment:
Intentional that the actual error message is not part of the snapshot any more?

Reply:
Not part of the snapshot anymore